incubator/jaeger/templates/collector-deploy.yaml

```diff
@@ -1,28 +1,29 @@
+{{- if .Values.collector.enabled -}}
 apiVersion: extensions/v1beta1
```

Comment:
A quick note, Deployments as extensions/v1beta1 will be removed from Kubernetes soon. Since this there has been `apps/v1beta1`, `apps/v1beta2`, and `apps/v1` API versions. For broad compatibility we'd like to see charts use `apps/v1beta2` for the moment. In k8s 1.10 the API versions prior to that can be removed.

This is just a note. Not going to have it hold up this pull request.

Reply:
Thank you! I'll make sure I update it when 1.10 is out.

Reply:
@mattfarina should I remove `extensions/v1beta1` now that k8s 1.10 is out?